vault/request_handling.go

```diff
@@ -1180,6 +1182,11 @@ func (c *Core) RegisterAuth(ctx context.Context, tokenTTL time.Duration, path st
 		Type:           auth.TokenType,
 	}
 
+	if te.TTL == 0 && (len(te.Policies) != 1 || te.Policies[0] != "root") {
```

Comment:
Here, for instance.

We could use auth.TTL and auth.TokenPolicies instead but it seems neater/nicer to source from the canonical TE at that point since it does exist/isn't nil.

Reply:
That's Core.RegisterAuth, not ExpirationManager.RegisterAuth.

Reply:
Sorry, linked to the wrong place.